WebRoot/js/zimbraMail/abook/model/ZmContact.js

```diff
@@ -427,7 +427,7 @@ function(node, args) {
 		}
 		AjxUtil.hashUpdate(contact.attr, node._attrs);	// merge new attrs just in case we don't have them
 		contact.list = args.list || new ZmContactList(null);
-		contact._list = {};
+		contact._list = contact._list || {};
```

Comment:
We were flushing the contact._list array which contains lists in which contact is present.